### DISCUSSION TITLE
Best Approach for Inline Table Edit with React Form Hook.

### DISCUSSION THREAD

**Discussion by CodAffection** [react-hook-form/react-hook-form#10817] [Q&A]
I'm constructing a food delivery order form. It's like master-detail form. where customer details are the master information and the list of selected food items is the details.

Planning to use a Dynamic array for the list of selected food items, which is to be rendered in an inline editable table. Is there any similar sample project that already exists? It's obvious to have a variable to store current edit row say _editIndex_, should I use react state to store its value or can I use react hook from the library?

Thanks in advance.

**Comment by lamhungypl** [comment]
Hi @CodAffection  , have you find the solution yet?
